app/Helpers/DBHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\DB;
+
+class DBHelper
+{
+    public static function getTables()
+    {
+        return DB::table('information_schema.tables')
+            ->select('table_name')
+            ->where('table_schema', '=', static::getDefaultConnectionConfiguration('database'))
```

Comment:
You can do: `DB::connection()->getDatabaseName()` to get the database name.

Reply:
:+1: